data/settings_mapping.json

```diff
@@ -168,10 +168,10 @@
             "shuffle_grotto_entrances",
             "shuffle_dungeon_entrances",
             "shuffle_bosses",
+            "spawn_positions",
```

Comment:
What's the reasoning behind this change? I think the previous grouping with the other one-way ERs makes more sense.

Reply:
It was more for aesthetics because it looked weird imo with a Combobox in the middle of a bunch of Checkboxes. It could go either way though for sure. Here's a picture for reference:

[https://media.discordapp.net/attachments/863163347654279198/1027772151513743461/unknown.png?width=1244&height=671](https://media.discordapp.net/attachments/863163347654279198/1027772151513743461/unknown.png?width=1244&height=671)

Reply:
We already have a mix of checkboxes and comboboxes mixed together in the two-way ER settings. I think it's more important to have sensible grouping.

Reply:
no worries I'll make the change really quick